bin/out

```diff
@@ -44,6 +44,39 @@ start_job() {
     # TODO make sure job is started
 }
 
+delete_resource() {
+    RESOURCE_TYPE=$1
+    RESOURCE=$2
+
+    [ -n "$RESOURCE_TYPE" ] || exit 1
+    [ -n "$RESOURCE" ] || exit 1
```

Comment:
Yeah don't worry it's still on a tab, half read but didn't sink in. Just been catching up with all the ones I opened with the jetstack guys.